xml/System.ComponentModel/RunWorkerCompletedEventArgs.xml

```diff
@@ -26,6 +26,8 @@
       <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
+ This property is the object provided in the DoWorkEventArgs.Result member of the DoWork event.
```

Comment:
This isn't accurate. RunWorkerCompletedEventArgs is a type rather than a property, and it's not provided in the DoWorkEventArgs.Result member. I'd propose instead the following: A **RunWorkerCompletedEventArgs** instance is provided to the \<xref:System.ComponentModel.RunWorkerCompletedEventHandler> delegate when the \<xref:System.ComponentModel.RunWorkerCompleted> event is fired.

Reply:
Yes, I apologize for not thinking. I wish everyone writing documentation were as careful, there are some very serious problems with some VB documentation. Go ahead and make any improvents to what I submitted, I will try to figure out what I need to do.

Reply:
I apologize if I misunderstand but if I understand then the suggested version is incomplete. Saying that an instance of RunWorkerCompletedEventArgs is provided does not specify where it is provided from. There should be something connecting the DoWork and the RunWorkerCompleted. I don't know if RunWorkerCompletedEventArgs is used anywhere else but if it is then that is the problem. It is difficult to say where the RunWorkerCompletedEventArgs comes from if it can come from multiple places.

In DoWork we have DoWorkEventArgs.Result that is then provided as RunWorkerCompletedEventArgs.Result. The two should be tied somehow. Even if it is part of the example then that would help. But the example is another frustrating example of being just a token example that does not help. It is my opinion however that the description of something should be complete enough that the example is not necessary, and that is rare in Microsoft documentation. Microsoft's typical solution is to make samples so complete that the relevant example is hidden among other stuff and that is where the name of my website SimpleSamoles.info comes from. The example should be relatively simple showing that the DoWork result is provided as the RunWorkerCompleted result.